msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md

```diff
@@ -290,6 +299,90 @@ async def _mention_activity(self, turn_context: TurnContext):
+
+> [!NOTE]
+>
+> * Tag mentions are supported in Teams desktop and web clients.
+> * Tag mentions are supported in GCC and GCC-H tenants only.
+
+##### Mention tags in a text message
+
+In the `mention.properties` object, add the property `'type': 'tag'`. If the property `'type': 'tag'` isn't added, the bot treats the mention as a user mention.
+
+Example:
+
+```javascript
+​var mention = new ChannelAccount(tagId, "Test Tag"); 
+​mention.Properties = JObject.Parse("{'type': 'tag'}"); 
+​var mentionObj = new Mention 
+​{ 
+​    Mentioned = mention, 
+​    Text = "<at>Test Tag</at>" 
+​}; 
+
+​var replyActivity = MessageFactory.Text("Hello " + mentionObj.Text); 
+​replyActivity.Entities = new List<Microsoft.Bot.Schema.Entity> { mentionObj }; 
+​await turnContext.SendActivityAsync(replyActivity, cancellationToken); 
+```
+
+##### Mention tags in an Adaptive Card
+
+In the `mentioned` object, add the property `"type": "tag"` in the Adaptive Card schema.  If the property `"type": "tag"` isn't added, the bot treats the mention as a user mention.
+
+You can get the list of the tags available in the channel using the [List teamworkTags](/graph/api/teamworktag-list?view=graph-rest-1.0&tabs=http&preserve-view=true) API.
+
+Example:
+
+```json
+​{ 
+​    "type": "mention", 
+    ​"text": "<at>my tag</at>", 
+​    "mentioned": { 
+            ​"id": "base64 encoded id" ,// tag graph 64 base ID
+​            "name": "my tag", 
+            ​"type": "tag" 
+​    } 
+​} 
+```
+
+###### Query Parameters
+
+|Name |Description |
+|---------|----------------|
+|`type`| The type of mention. The supported type is `tag`. The tag format is base 64 encoded ID​. For example, `NTI4ZGJlM2YtMTVlMC00ZTM3LTg0YTEtMDBjYzMwNTg0N2RkIyNlYzgwMTVmMC1iMmYxLTQxZTItODA0OC1hMGE2OTcwNmM5ZGIjI3RxRE04YndyVQ==​`.
+
+###### Error code
+
+| Status code | Error code | Message values | Retry request | Developer action|
+|----------------|-----------------|-----------------|----------------|----------------|
+| 400 | **Code**: `Bad Request` | ​Mentioned tag with ID {id string} doesn't exist in current team<br/>​Tag can only be mentioned in channel<br/>Invalid mentioned tag because no tag exists in the team| No | Reevaluate request payload for errors. Check returned error message for details. |
+| 502 | **Code**: `Bad Gateway` | Invalid team group ID<br/> ​Malformed tenant ID for the tag<br/> ​Mention ID can't be resolved | No |Retry manually.|
+
+##### Throttling limits
+
+Any request can be evaluated against multiple limits, depending on the scope, the window type (short and long), number of tags per message, and other factors. The first limit to be reached triggers throttling behavior.
+
+Ensure that you don't exceed the throttling limits to avoid heavy traffic to the notification service and  the IC3 service. For example, A bot can send only two messages with tags mention in a five-second window and each message can have only up to 10 tags.
```

Comment:
Let's rephrase the words to something below or similar:
Ensure that you don't exceed the throttling limits to avoid failed message delivery. For example, xxxxxxx

Reply:
Updated.